internal/dockerauth/api.go

```diff
@@ -58,40 +53,17 @@ func parseScope(s string) ([]resourceAccessRights, error) {
 	if s == "" {
 		return nil, nil
 	}
-	var errs []error
 	var ras []resourceAccessRights
 	for _, s := range strings.Split(s, " ") {
 		ra, err := parseResourceAccessRights(s)
 		if err != nil {
-			errs = append(errs, err)
-			continue
+			return nil, errgo.Notef(err, "invalid access rights %q", s)
```

Comment:
"invalid scope" makes more sense as that's the parameter name.

Reply:
done